boards/nxp/frdm_mcxn236/frdm_mcxn236.dts

```diff
@@ -131,6 +131,10 @@
 	status = "okay";
 };
 
+&i3c1 {
```

Comment:
Add I3C to the board YAML as well, like here: https://github.com/zephyrproject-rtos/zephyr/blob/e1389fa2cff66c4f0c1ac5d0b1b932ef0e3faf06/boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.yaml#L31

Also, does this board have any sensor on the I3C bus we could enable here, to provide an easy way to validate the bus works?

Reply:
Hi @danieldegrasse , frdmmcxn236 board don't have I3C sensor.

Reply:
FYI, some Freedom boards include the P3T1755 I3C temperature sensor.  Mahesh just enabled this sensor in [Add I3C support for FRDM-MCXN947](https://github.com/zephyrproject-rtos/zephyr/pull/79738).  The N947 is a very similar SOC and shares the same I3C peripheral, so the driver is validated on that board.

For other boards without this sensor, NXP offers the [P3T1755DP-ARD Arduino shield](https://www.nxp.com/design/design-center/development-boards-and-designs/analog-toolbox/arduino-shields-solutions/p3t1755dp-arduino-shield-evaluation-board:P3T1755DP-ARD).  @hakehuang has one of these shields so we can test boards without I3C sensors if needed.

Best regards

Reply:
in P3T1755, the pin on arduino is lay as below: is it possible to align? as I can see the P1_11 is not compatible
```
![image](https://github.com/user-attachments/assets/08bd8141-8367-4f10-8d50-a054cfea3f87)

```